src/GitHub.Api/Git/RepositoryManager.cs

```diff
@@ -12,22 +12,21 @@ public interface IRepositoryManager : IDisposable
         event Action<IUser> OnGitUserLoaded;
         event Action<bool> OnIsBusyChanged;
         event Action<string> OnLocalBranchAdded;
-        event Action<Dictionary<string, ConfigBranch>> OnLocalBranchListUpdated;
+        event Action<IDictionary<string, ConfigBranch>> OnLocalBranchListUpdated;
```

Comment:
Why the change to the interface?

Reply:
Hmm. It wasn't necessary. Rolling it back

Reply:
Yea, I originally made this change because I was trying to figure out serializations and
`Dictionary<string, Dictionary<string, ConfigBranch>>>` != `IDictionary<string, IDictionary<string, ConfigBranch>>>`